src/rules.js

```diff
@@ -76,9 +76,9 @@ function breakOutRuleName(ruleName) {
     if (scope || plugin) {
         if (scope && plugin) {
             pluginName = `${scope}/${plugin}`;
+        } else {
+            pluginName = scope || plugin;
         }
-
-        pluginName = scope || plugin;
```

Comment:
Thank you for this fix.  I must have been thinking about a return not needing an else.

Reply:
Great! 🎉  Thank you for your quick feedback!

Could this go in as a patch or with the next release? I would love to recommend **LintLens** to the users of our plugin, but for the rule descriptions to properly show this fix is would be necessary.

Reply:
Just got v3.0.2, thanks! 😄

Reply:
No problem.  Thanks for your contribution.